pages/learn/getting-started.md

```diff
@@ -177,10 +177,11 @@ Your application will then be downloaded and executed by all the devices you hav
 
 ## Next steps
 
+- To easily find, deploy and publish your applications for balena supported devices, head to [balenaHub](https://hub.balena.io)
```

Comment:
```suggestion
- To easily find, deploy and publish your applications for {{ $names.company.lower }} supported devices, head to [balenaHub](https://hub.balena.io)
```
Purely nitpick, otherwise LGTM
On the word, `supported devices` we can embed this link - https://www.balena.io/docs/reference/hardware/devices/

Reply:
Actually let me add that.